src/App.vue

```diff
@@ -128,33 +128,53 @@ export default {
 	},
 
 	methods: {
-		loadNotes() {
-			fetchNotes()
-				.then(data => {
-					if (data === null) {
-						// nothing changed
-						return
-					}
-					if (data.notes !== null) {
-						this.error = false
-						this.routeDefault(data.lastViewedNote)
-					} else if (this.loading.notes) {
-						// only show error state if not loading in background
-						this.error = data.errorMessage
-					} else {
-						console.error('Server error while updating list of notes: ' + data.errorMessage)
-					}
-				})
-				.catch(() => {
+		async loadNotes() {
+			console.log('[App.loadNotes] Starting initial load')
+			// Skip refresh if in search mode - search results should not be overwritten
+			const searchText = store.state.app.searchText
+			if (searchText && searchText.trim() !== '') {
+				console.log('[App.loadNotes] Skipping - in search mode with query:', searchText)
+				this.startRefreshTimer(config.interval.notes.refresh)
+				return
+			}
```

Comment:
Plus, why have we removed the separation of notes into "days" like  "Today", "Yesterday"? Now, we don't have it:

<img width="1182" height="320" alt="Image" src="https://github.com/user-attachments/assets/61e27cce-84c8-4d66-9ca0-e760d205ef81" />